index.js

```diff
@@ -100,10 +100,24 @@ class K8sVMExecutor extends Executor {
             launcher_version: this.launchVersion,
             base_image: this.baseImage
         });
+
+        const podConfig = yaml.safeLoad(podTemplate);
+
+        const tolerations = hoek.reach(config, 'tolerations', { default: [] });
```

Comment:
This is cluster-level config, not user-config right? https://github.com/screwdriver-cd/screwdriver/pull/828/files

So this value should be read from the constructor, line 63:
`this.tolerations = hoek.reach(options, 'kubernetes.tolerations', { default: [] });`